File: weather-demo-app-apple/Weather/Domain/Model/CurrentWeather.swift
//
//  CurrentWeather.swift
//  Weather
//
//  Created by Onur on 4/11/26.
//

struct CurrentWeather: Equatable {
    let city: String
    let condition: WeatherCondition
    let description: String
    let temperatureCelsius: Double
    let feelsLikeCelsius: Double
    let humidityPercent: Int
    let windSpeedKph: Double
    let iconCode: String
}

File: weather-demo-app-android/app/src/main/java/com/demo/weather/domain/model/CurrentWeather.kt
package com.demo.weather.domain.model

data class CurrentWeather(
    val city: String,
    val condition: WeatherCondition,
    val description: String,
    val temperatureCelsius: Double,
    val feelsLikeCelsius: Double,
    val humidityPercent: Int,
    val windSpeedKph: Double,
    val iconCode: String
)